src/ray/raylet/node_manager.cc

```diff
@@ -223,13 +223,14 @@ void NodeManager::HeartbeatAdded(gcs::AsyncGcsClient *client, const ClientID &cl
   }
   // Locate the client id in remote client table and update available resources based on
   // the received heartbeat information.
-  if (this->cluster_resource_map_.count(client_id) == 0) {
+  auto it = this->cluster_resource_map_.find(client_id);
+  if (it == cluster_resource_map_.end()) {
```

Comment:
the `.count() == 0` is supposed to be idiomatically equivalent. Do you find your proposed way better in performance?

Reply:
count() means we need try to find a item in the map. and cluster_resource_map_[client_id] will find it again. Although find is O(n) operation for unordered_map, but calculation hash and function call will have cost, right. The fix will change find operation from twice to once